xml/System.Threading.Channels/ChannelReader`1.xml

```diff
@@ -85,8 +85,8 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets whether <see cref="M:System.Threading.Channels.ChannelReader`1.TryPeek(`0@)" /> is available for use on this <see cref="T:System.Threading.Channels.ChannelReader`1" /> instance.</summary>
```

Comment:
@gewarren First time I see `0@` as an argument. What does it mean?

Reply:
I think `` `0`` means the typeparam and `@` means out so it means `out T` 

Anyway your tool wrote it @carlossanlop 😄

Reply:
This is the method signature:

```
public virtual bool TryPeek([MaybeNullWhen(false)] out T item)
```

I think ``0` refers to the type parameter and `@` refers to it being an `out` parameter. @TianqiZhang Is that correct?

Reply:
> Anyway your tool wrote it @carlossanlop 😄

Really? It might be a recent bug fix, maybe introduced by Jeff or Eirik, who have also improved the tool.

Reply:
Isn't the tool just using the DocID?

```
<MemberSignature Language="DocId" Value="M:System.Threading.Channels.ChannelReader`1.TryPeek(`0@)" />
```